src/Models/Apis.ts

```diff
@@ -4,22 +4,25 @@
 import * as vscode from "vscode";
 
 import { AzureAccount } from "../azure-account.api";
-import { AzureAccountCommands } from "../common/Commands";
+import { AzureAccountCommands, VscodeCommands } from "../common/Commands";
 import { DependentExtensionNotFoundError } from "../common/Error/OperationFailedErrors/DependentExtensionNotFoundError";
-
-import { ExtensionName } from "./Interfaces/Api";
+import { ExtensionName, DependentExtensions, ExtensionNameIdMap } from "./Interfaces/Api";
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 export function getExtension(name: ExtensionName): any {
+  // }
   switch (name) {
-    case ExtensionName.Toolkit: {
-      const toolkit = vscode.extensions.getExtension("vsciot-vscode.azure-iot-toolkit");
-      return toolkit ? toolkit.exports : undefined;
-    }
     case ExtensionName.AzureAccount: {
-      const azureAccount = vscode.extensions.getExtension<AzureAccount>("ms-vscode.azure-account");
+      const azureAccount = vscode.extensions.getExtension<AzureAccount>(DependentExtensions.azureAccount);

```

Comment:
could AzureAccount behavior the same as other extensions?

Reply:
I tested and it is okay to remove the type defined for AzureAccout. Removed the type restriction.